src/Analysis/Ast/Impl/Types/ArgumentSet.cs

```diff
@@ -178,7 +179,7 @@ internal sealed class ArgumentSet : IArgumentSet {
                     if (formalParamIndex >= overload.Parameters.Count) {
                         // We ran out of formal parameters and yet haven't seen
                         // any sequence or dictionary ones. This looks like an error.
-                        _errors.Add(new DiagnosticsEntry(Resources.Analysis_TooManyFunctionArguments, arg.GetLocation(eval).Span,
+                        _errors.Add(new DiagnosticsEntry(Resources.Analysis_TooManyFunctionArguments, callLocation.Span,
```

Comment:
Added this because currently on too many function arguments we highlight one of the extra args then bail. We should just highlight the whole function instead.